apps/site/src/pages/api/rewrites/sandboxed-block-demo.api.ts

```diff
@@ -40,8 +40,12 @@ const handler: NextApiHandler = async (req, res) => {
 
   const mockBlockDockVersion = packageJson.dependencies["mock-block-dock"];
 
-  const reactVersion =
-    blockMetadata.externals?.react ?? packageJson.dependencies.react;
+  // // @todo restore this when esm.sh treats version ranges correctly
+  // //   it current pulls non-latest versions of packages if a range is supplied
+  // const reactVersion =
+  //   blockMetadata.externals?.react ?? packageJson.dependencies.react;
+
```

Comment:
```suggestion
  // @todo restore this when esm.sh treats version ranges correctly
  //    it currently pulls non-latest versions of packages if a range is supplied
  // const reactVersion =
  //   blockMetadata.externals?.react ?? packageJson.dependencies.react;

```

Reply:
This is intentional, so that you can select the whole block and uncomment it, and still be left with a comment and code. Maybe it's a bit pointless but it's not doing any harm and isn't a formatting error

Reply:
I'm a bit confused. When you restore it you want the comment still there?

Reply:
You want it to look like
```
  // @todo restore this when esm.sh treats version ranges correctly
  //   it current pulls non-latest versions of packages if a range is supplied
  const reactVersion =
    blockMetadata.externals?.react ?? packageJson.dependencies.react;
```

once it's uncommented?

Reply:
If so, what is the "restore this" referring to

Reply:
I have applied your suggestion, thank you